model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -169,7 +169,7 @@ class LicenseInfoResolver(
     ): DetectedLicenseInfo {
         val filteredFindings = findings.map { finding ->
             val (copyrightGarbage, copyrightFindings) = finding.copyrights.partition { copyrightFinding ->
-                copyrightFinding.statement in copyrightGarbage.items
+                copyrightGarbage.items.any { copyrightFinding.statement.startsWith(it) }
```

Comment:
This change does not only allow copyright garbage entries to be prefixes, but treats all copyright garbage entries as prefixes. I'm a bit uncertain whether this is too risky, but don't have a good alternative yet...

Reply:
I remember the discussion we had a long time ago about deliberately not supporting wildcards in garbage, and I'm still fine with not supporting them (or regexes). Interpreting all garbage as prefixes was the least intrusive thing I could think of to solve the case described in https://github.com/oss-review-toolkit/ort/discussions/6580, so I thought I'd give it a try. After all, this is anyway just a temporary solution to me until we hopefully at some point can properly [curate copyrights](https://github.com/oss-review-toolkit/ort/issues/4519). And I believe it's quite unlikely to break anything.

Reply:
> After all, this is anyway just a temporary solution 

What are you trying to say with this?

> it's quite unlikely to break anything

I think it adds risk to match entries which have not been foreseen.
My gut feeling is that this risk some users wouldn't want to take without need.
Maybe this a good one for a group discussion?

Reply:
> What are you trying to say with this?

It's written in the same sentence: In the mid-term I'm expecting the current Copyright garbage approach to be replaced with a Copyright finding curation mechanism that allow to map findings to `NONE` to make them disappear, similar to like with licenses.

> Maybe this a good one for a group discussion?

You mean in person? For a start, feel free to join the discussion at https://github.com/oss-review-toolkit/ort/discussions/6580.

PS: I'm personally completely fine with not merging this. I was just trying to help one of our users with a quick and simple fix.

Reply:
> You mean in person? For a start, feel free to join the discussion at #6580.

I meant, maybe in Ort call thursday or monday. Anyhow, I'll join discussion, seems not too urgent so give me a night to sleep about it.

Reply:
> give me a night to sleep about it.

I've meanwhile taken the `Regex` approach that we discussed.

Reply:
> > You mean in person? For a start, feel free to join the discussion at #6580.
> 
> I meant, maybe in Ort call thursday or monday. Anyhow, I'll join discussion, seems not too urgent so give me a night to sleep about it.

Could you please brief here the outcome of the discussion?

Reply:
@dgutson, see my reply just one comment above yours:

> I've meanwhile taken the `Regex` approach that we discussed.

Reply:
@sschuberth because I saw it, I asked :) I recall you saying that not allowing regexes was a deliberate decision, so I wanted to know the rationale behind this change.

Reply:
> I recall you saying that not allowing regexes was a deliberate decision, so I wanted to know the rationale behind this change.

Yes, we deliberately decided to not use regexes as the primary / only way to define garbage statements. However, I'm now proposing a new section in the garbage file for regexes. That's a compromise @fviernau indicated to be fine with.